### PR TITLE
add deprecated warnings

### PR DESCRIPTION
**Pull request by commy2**
ref: #2012

### CODE REVIEW

**Comment by jonpas** [comment]
Version 1.5?

**Comment by commy2** [comment]
oops. 3.5

**Comment by jonpas** [comment]
:+1: Best would be to write `3.5.0` IMO, to prevent confusion.

File: addons/main/script_macros.hpp [hunk 1]
```diff
@@ -132,7 +132,7 @@
 #define ACE_LOGWARNING_7(message,arg1,arg2,arg3,arg4,arg5,arg6,arg7) ACE_LOGWARNING(FORMAT_7(message,arg1,arg2,arg3,arg4,arg5,arg6,arg7))
 #define ACE_LOGWARNING_8(message,arg1,arg2,arg3,arg4,arg5,arg6,arg7,arg8) ACE_LOGWARNING(FORMAT_8(message,arg1,arg2,arg3,arg4,arg5,arg6,arg7,arg8))
 
-#define ACE_WARNINGFORMAT(message) ACE_LOGFORMAT(COMPONENT,"WARNING",message)
+#define ACE_WARNINGFORMAT(message) ACE_LOG(COMPONENT,"WARNING",message)
```
Comment (PabstMirror) [inline addons/main/script_macros.hpp]: I think
`ACE_LOGWARNING` is for rpt writing
`ACE_WARNINGFORMAT` just returns a string
so I don't think we wanto change it

**Comment by commy2** [comment]
Example:
`21:48:03 [ACE] (common) WARNING: ace_common_fnc_isTurnedOut is deprecated. Support will be dropped in version 3.5.0. Replaced by: isTurnedOut`

File: addons/main/script_macros.hpp (hunk 1 above)
Comment (commy2) [inline addons/main/script_macros.hpp]: Weird. Makes sense though

**Comment by commy2** [comment]
done

**Comment by MikeMatrix** [comment]
Maybe replace it by an deprecation macro?
Would clean it up a little as well as introduce a standard way of deprecating things.

**Comment by commy2** [comment]
Agreed and done

**Comment by jonpas** [comment]
Doesn't CBA already have deprecation macros?

**Comment by commy2** [comment]
Well, they aren't ACE formated.

**Comment by jonpas** [comment]
True.

**Comment by MikeMatrix** [comment]
One other thing might to be considered is to use `QFUNC` instead of writing the full function name? Just a thought.

**Comment by commy2** [comment]
These will be removed soonish anyway and will probably never be moved again.

**Comment by MikeMatrix** [comment]
True. Maybe for future use might be easier.

**Comment by commy2** [comment]
k. Anything else or merge?

**Comment by MikeMatrix** [comment]
Looks good to me.
:+1:

**Pull request merged**